src/utils/QuerySummaryUtils.ts

```diff
@@ -50,7 +48,7 @@ export class QuerySummaryUtils {
   }
 
   private static isInfiniteScrollMode(root: HTMLElement) {
-    const resultListSelector = `.${Component.computeCssClassNameForType('ResultList')}`;
+    const resultListSelector = `.CoveoResultList`;
```

Comment:
This line here was causing some nasty circular dependency issue...

Reply:
I am guilty of introducing this one. I have been more attentive since and will continue looking out for them in the future,